CHANGELOG.md

```diff
@@ -3,6 +3,9 @@
 - Diagnostics related feature requests and improvements [#5951](https://github.com/dotnet/vscode-csharp/issues/5951)
 - Debug from .csproj and .sln [#5876](https://github.com/dotnet/vscode-csharp/issues/5876)
 
+# Latest
```

Comment:
this section can be removed, and put in the one below.  sections are autogenerated now

Reply:
updated